_posts/digests/2019-09-22-32.md

```diff
@@ -0,0 +1,128 @@
+
+* [Facial recognition tech is a problem. Here's how the Democratic candidates plan to tackle it.](https://www.vox.com/future-perfect/2019/8/21/20814153/facial-recognition-ban-bernie-sanders-elizabeth-warren-kamala-harris-julian-castro-cory-booker) - As the growing use of facial recognition technology draws criticism from the public, Democratic presidential candidates are starting to articulate how they'd handle the tech if they're elected. Some candidates are staking out stronger positions than others. In August, Sen.
+
+* [Towards Fairer Datasets: Filtering and Balancing the Distribution of the People Subtree in the ImageNet Hierarchy](http://image-net.org/update-sep-17-2019.php) - Kaiyu Yang (Princeton University), Klint Qinami (Princeton University), Li Fei-Fei (Stanford University), Jia Deng (Princeton University), Olga Russakovsky (Princeton University) As AI technology advances from research lab curiosities into people's daily lives, ensuring that AI systems produce appropriate and fair results has become an important scientific question.
+
+* [Rep. Takano Introduces the Justice in Forensic Algorithms Act to Protect Defendants’ Due Process Rights in the Criminal Justice System](https://takano.house.gov/newsroom/press-releases/rep-takano-introduces-the-justice-in-forensic-algorithms-act-to-protect-defendants-due-process-rights-in-the-criminal-justice-system) - Today, Rep. Mark Takano (D-Calif.) introduced the Justice in Forensic Algorithms Act of 2019 to ensure that defendants have access to source code and other information necessary to exercise their confrontational and due process rights when algorithms are used to analyze evidence in their case.
+
+* [Excavating AI](https://www.excavating.ai) - You open up a database of pictures used to train artificial intelligence systems. At first, things seem straightforward. You're met with thousands of images: apples and oranges, birds, dogs, horses, mountains, clouds, houses, and street signs.
+
+### Expert Opinions & Discussion within the field
+
+* [Emergent Tool Use from Multi-Agent Interaction](https://openai.com/blog/emergent-tool-use/) - We've observed agents discovering progressively more complex tool use while playing a simple game of hide-and-seek.
+
+* [A Step Toward Quantifying Independently Reproducible Machine Learning Research](https://arxiv.org/abs/1909.06674v1) - Abstract: What makes a paper independently reproducible? Debates on reproducibility center around intuition or assumptions but lack empirical results. Our field focuses on releasing code, which is important, but is not sufficient for determining reproducibility. We take the first step toward a quantifiable answer by manually attempting to implement 255 papers published from 1984 until 2017, recording features of each paper, and performing statistical analysis of the results. For each paper, we did not look at the authors code, if released, in order to prevent bias toward discrepancies between code and paper.

```

Comment:
remove - let's not add primary srcs unless it's a blog post or something more easily understandable.